packages/editor/src/client/blocks/htmlBlock.tsx

```diff
@@ -19,6 +20,12 @@ export const HTMLBlock = ({value, onChange, autofocus}: BlockProps<HTMLBlockValu
     }
   }, [])
 
+  useEffect(() => {
+    executeScriptElements(
+      document.querySelectorAll('div[data-html-block]') as NodeListOf<HTMLElement>
```

Comment:
This will execute them multiple times now when you have multiple HTML block.
We should use a ref, attach it to the div and do `<ref>.querySelector`